docs/specs/Mountain State Adventure Destination/SPEC-08-adventure-card-component/tasks.md

```
@@ -0,0 +1,348 @@
+                    │  ┌──────────┐         │
+                    │  │ Task 4.1 │         │
+                    │  │Grid Wire │         │
+                    │  └──────────┘         │
+                    │       │               │
+                    │       ▼               │
+                    │  ┌──────────┐         │
+                    │  │ Task 4.2 │         │
+                    │  │  Test    │         │
+                    │  └──────────┘         │
+                    └───────────────────────┘
+```
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)